<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-09-12T17:06:28 BDT" name="MobileAutomationSuite" finished-at="2025-09-12T17:07:05 BDT" duration-ms="36386">
    <groups>
    </groups>
    <test started-at="2025-09-12T17:06:28 BDT" name="Appium Tests" finished-at="2025-09-12T17:07:05 BDT" duration-ms="36386">
      <class name="tests.OpenToHomeTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:tests.OpenToHomeTest@5ddeb7cb]" started-at="2025-09-12T17:06:28 BDT" name="setUp" finished-at="2025-09-12T17:06:38 BDT" duration-ms="9926" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@38320e34]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testOpenAppToHome()[pri:0, instance:tests.OpenToHomeTest@5ddeb7cb]" started-at="2025-09-12T17:06:38 BDT" name="testOpenAppToHome" finished-at="2025-09-12T17:06:57 BDT" duration-ms="18416" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[❌ Name Page did not open. expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: ❌ Name Page did not open. expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at tests.OpenToHomeTest.enterName(OpenToHomeTest.java:73)
at tests.OpenToHomeTest.testOpenAppToHome(OpenToHomeTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testOpenAppToHome -->
        <test-method is-config="true" signature="attachScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:tests.OpenToHomeTest@5ddeb7cb]" started-at="2025-09-12T17:06:57 BDT" name="attachScreenshotOnFailure" finished-at="2025-09-12T17:06:57 BDT" duration-ms="632" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testOpenAppToHome status=FAILURE method=OpenToHomeTest.testOpenAppToHome()[pri:0, instance:tests.OpenToHomeTest@5ddeb7cb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- attachScreenshotOnFailure -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.OpenToHomeTest@5ddeb7cb]" started-at="2025-09-12T17:06:57 BDT" name="tearDown" finished-at="2025-09-12T17:06:58 BDT" duration-ms="774" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.OpenToHomeTest -->
      <class name="tests.LaunchAppTest">
        <test-method is-config="true" signature="setUp(org.testng.ITestContext)[pri:0, instance:tests.LaunchAppTest@7c37508a]" started-at="2025-09-12T17:06:58 BDT" name="setUp" finished-at="2025-09-12T17:07:03 BDT" duration-ms="5248" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@38320e34]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testLaunchApp()[pri:0, instance:tests.LaunchAppTest@7c37508a]" started-at="2025-09-12T17:07:03 BDT" name="testLaunchApp" finished-at="2025-09-12T17:07:04 BDT" duration-ms="899" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLaunchApp -->
        <test-method is-config="true" signature="attachScreenshotOnFailure(org.testng.ITestResult)[pri:0, instance:tests.LaunchAppTest@7c37508a]" started-at="2025-09-12T17:07:04 BDT" name="attachScreenshotOnFailure" finished-at="2025-09-12T17:07:04 BDT" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLaunchApp status=SUCCESS method=LaunchAppTest.testLaunchApp()[pri:0, instance:tests.LaunchAppTest@7c37508a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- attachScreenshotOnFailure -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.LaunchAppTest@7c37508a]" started-at="2025-09-12T17:07:04 BDT" name="tearDown" finished-at="2025-09-12T17:07:05 BDT" duration-ms="401" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.LaunchAppTest -->
    </test> <!-- Appium Tests -->
  </suite> <!-- MobileAutomationSuite -->
</testng-results>
